src/test/java/redis/clients/jedis/tests/commands/AllKindOfValuesCommandsTest.java

```diff
@@ -615,6 +616,29 @@ public void scanCount() {
   }
 
   @Test
+  public void scanIsCompleteIteration() {
+    for (int i = 0; i < 100; i++) {
+      jedis.set("a" + i, "a" + i);
+    }
+
+    ScanResult<String> result = scanCompletely();
```

Comment:
Well, there we are again at the problem, that *in theory* I cannot test this reliably ;-)
But nevertheless I added it